extensions/llama/python/llama.py

```diff
@@ -48,15 +45,30 @@ async def run_inference(
     ) -> ExecuteResult:
         resolved = resolve_prompt(prompt, parameters, aiconfig)
 
-        llm = Llama(self.model_path)
-        response = llm(resolved)
+        # print(f"{prompt=}, {resolved=}, {parameters=}, {options=},{aiconfig=}")
 
-        try:
-            texts = [r["choices"][0]["text"] for r in response]
-        except TypeError:
-            texts = [response["choices"][0]["text"]]
+        stream = True
+
+        llm = Llama(self.model_path)
+        acc = ""
+        if stream:
+            for res in llm(resolved, stream=True):
+                raw_data = res["choices"][0]
+                data = raw_data["text"]
+                index = int(raw_data["index"])
+                acc += data
+                if options:
+                    options.stream_callback(data, acc, index)
+            print(flush=True)
+            return ExecuteResult(output_type="execute_result", data=[acc], metadata={})
+        else:
+            response = llm(resolved)
+            try:
+                texts = [r["choices"][0]["text"] for r in response]
+            except TypeError:
+                texts = [response["choices"][0]["text"]]
 
-        return ExecuteResult(output_type="execute_result", data=texts, metadata={})
+            return ExecuteResult(output_type="execute_result", data=texts, metadata={})
```

Comment:
`texts` is an array of strings here, right? If so, should update `get_output_text` to join the data to a single string?

Reply:
Good question - what is it supposed to return if there's more than one string? cc @LastMileNewGrad